src/fake_useragent/get_version.py

```diff
@@ -0,0 +1,8 @@
+"""Simple helper file to retrieve the version."""
+
+try:
+    from importlib import metadata
+
+    __version__ = metadata.version("fake-useragent")
+except ImportError:
```

Comment:
Doesn't this raise a [`PackageNotFoundError`](https://docs.python.org/3/library/importlib.metadata.html#importlib.metadata.version)?

Reply:
No an import error in case importlib can't be found.